docs/development.md

```diff
@@ -1,6 +1,260 @@
 # Development documentation
 
-## Writing integration tests
+- [Introduction](#introduction)
+  * [Directory overview](#directory-overview)
+  * [Package overview](#package-overview)
+  * [Build targets](#build-targets)
+  * [Unit tests](#unit-tests)
+- [Extending commander](#extending-commander)
+  * [Add a new field to the `YAML` suite](#add-a-new-field-to-the--yaml--suite)
+  * [Writing integration tests](#writing-integration-tests)
+
+
+## Introduction
+
+### Directory overview
+
+```
+├── cmd          # Contains the composition root and files which will be compiled to binaries.
+├── docs         # Documentation for commander
+├── examples     # Examples how to use commander with to give you a little bit inspiration.
```

Comment:
Examples **of** how to use commander ...

Just missing a little **of** in here